balancer/grpclb/grpclb.go

```diff
@@ -148,11 +148,11 @@ func (b *lbBuilder) Build(cc balancer.ClientConn, opt balancer.BuildOptions) bal
 	if opt.CredsBundle != nil {
 		lb.grpclbClientConnCreds, err = opt.CredsBundle.NewWithMode(internal.CredsBundleModeBalancer)
 		if err != nil {
-			grpclog.GRPCLB.Warningf("lbBalancer: client connection creds NewWithMode failed: %v", err)
```

Comment:
GRPCLB is an acronym, so should be all-caps.

Reply:
This has been replaced with an in-package defined "logger" var.